lib/controller.rb

```diff
@@ -0,0 +1,41 @@
+require 'web_human_player'
+require 'core/ttt/board'
+require 'core/ttt/game'
+
+class Controller
+
+  attr_reader :game
+
+  def initialize
+    @player_x = WebHumanPlayer.new("X")
+    @player_o = WebHumanPlayer.new("O")
+    @board = Board.new(Array.new(9,"-"))
+    @game = Game.new(@board, @player_x, @player_o)
+  end
+
+  def mark_board(current_board, index)
+    board = Board.new(current_board.split(""))
```

Comment:
I wonder if the board could benefit from some factory initializer methods to make it a bit easier to read.

so line 12 may have `@board = Board.empty_board)`
and line 17 may have `board = Board.with_moves(current_board.split(""))`

Having the controller be aware that the board needs to take in an array of 9 hyphens may be a bit too much knowledge here.